src/home/components/components/pokemon-detail/MissingPokemon.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import {useEffect, useState} from 'react'
+import {useHistory} from 'react-router-dom'
+
+interface IMissingPokemon {
+
+}
+
+export const MissingPokemon:React.FC<IMissingPokemon> = () => {
+    const history = useHistory()
+    const [time, setTime] = useState(5)
```

Comment:
Really nice logic for the countdown, and I like the effect, but also check my previous comment.
It would be better if page would actually load the content.